actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -140,17 +140,30 @@ def log_error(request, wrapper)
 
         message = []
         message << "  "
-        message << "#{wrapper.exception_class_name} (#{wrapper.message}):"
         if wrapper.has_cause?
-          message << "\nCauses:"
+          message << "#{wrapper.exception_class_name} (#{wrapper.message})"
           wrapper.wrapped_causes.each do |wrapped_cause|
-            message << "#{wrapped_cause.exception_class_name} (#{wrapped_cause.message})"
+            message << "Caused by: #{wrapped_cause.exception_class_name} (#{wrapped_cause.message})"
           end
+
+          message << "\nInfos for: #{wrapper.exception_class_name} (#{wrapper.message}):"
```

Comment:
```suggestion
          message << "\nInformation for: #{wrapper.exception_class_name} (#{wrapper.message}):"
```

Reply:
Changed both "Infos" and the tests accordingly.